streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java

```diff
@@ -68,6 +69,7 @@
 
 @Timeout(600)
 @Category({IntegrationTest.class})
+@Tag("integration")
```

Comment:
Do we need both the annotations here? The latter is used by JUnit5 and former is used by JUnit4. You can consult the build logic at the following place to check how we use these tags.
https://github.com/apache/kafka/blob/d481163d55a1cea3206fa6e386b24ac84e17c3d7/build.gradle#L473

Reply:
I saw the build logic in 'build.gradle' file and I was confused because it excluded two tags here, but in the same test folder (.../streams/integration/utils) some tests only have @Category(IntegrationTest.class) were excluded from unitTest while some were not. For example, AbstractJoinIntegrationTest.java and MetricsReporterIntegrationTest.java both have "@Category({IntegrationTest.class})" annotation but the former could be excluded from unitTest while the latter not. 

https://github.com/apache/kafka/blob/d481163d55a1cea3206fa6e386b24ac84e17c3d7/build.gradle#L517C1-L519C62

So I didn't remove it since I was not sure if it is used in other scopes.

Reply:
Hi, thank you for telling "The latter is used by JUnit5 and former is used by JUnit4". I did more investigation, now I understand the difference. We do not need both annotations here, will fix this.